subscription-service/src/main/kotlin/com/egm/stellio/subscription/model/GeoQuery.kt

```diff
@@ -10,5 +11,5 @@ data class GeoQuery(
     val coordinates: Any,
     @JsonIgnore
     val pgisGeometry: String?,
-    val geoproperty: String? = null
+    val geoproperty: String? = NGSILD_LOCATION_PROPERTY
```

Comment:
if you go for the expanded version (which seems good), ensure that the whole chain is ready for it. For instance:
- Is it expanded when you receive a subscription?
- Is it compacter when you return a subscription?
- ...

Reply:
When we receive a subcription, it is expanded and when we return subscription we compact them.

Reply:
are you sure of this? where is it done?

Reply:
In `Notification service` - `buildNotificationData` we compact entity

Reply:
In `EntityEventListenerService` -  `handleEntityEvent` we expand entity